hnn_core/network.py

```diff
@@ -679,6 +738,9 @@ def _create_drive_conns(self, target_populations, weights_by_receptor,
         src_gid_curr = self._n_gids
 
         if cell_specific:
+            _check_option('n_drive_cells', n_drive_cells,
+                          allowed_values=[self.n_cells],
+                          extra=' when cell_specific is True')
```

Comment:
say this in the docstring of `cell_specific`, that only `n_drive_cells=n_cells` is allowed for `cell_specific=True`

Reply:
Addressed in https://github.com/jonescompneurolab/hnn-core/pull/383/commits/1d00e7319e84da6855eef82f862e401243ae740a